docs/source/examples/demo_metrics_correlation.ipynb

```diff
@@ -0,0 +1,496 @@
+   "source": [
+    "M, T = df_pd.shape\n",
+    "cm = plt.get_cmap('tab20')\n",
+    "vals = np.linspace(0, 1, M)\n",
+    "\n",
+    "for i in range(M):\n",
+    "    plt.plot(df_pd.iloc[i], color = cm(vals[i]))\n",
+    "\n",
+    "plt.title('Example metrics from the Server Machine Dataset')\n",
+    "plt.ylabel('Normalized metrics value')\n",
+    "plt.xlabel('Timestamp')"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Step 3: Metrics correlation\n"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "The metrics correlation algorithm finds events in a set of metrics data, where an event is defined as a window in time where multiple metrics simultaneously display anomalous behavior. Given a time span and a total set of metrics, the metrics correlation algorithm automatically determines how many events occurred, when they occurred, and which metrics were involved in each event. This algorithm requires no prior training and minimal configuration.\n",
+    "\n",
+    "Let’s see how it works for the Server Machine Dataset example."
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "outputs": [],
+   "source": [
+    "input_json = {\"metrics\": df_pd.values.tolist()}"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Now we can run the metrics correlation algorithm on this data via the execute API:"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "outputs": [],
+   "source": [
+    "results = ml_client.execute(\n",
+    "    algorithm_name = \"METRICS_CORRELATION\",\n",
+    "    input_json = input_json\n",
+    ")"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "The results are given as a list, where each element of the list is an event that the algorithm has discovered in the data. If the length of the list is zero, then the algorithm has discovered no events in the given input data.\n",
+    "\n",
+    "In this case, the algorithm discovers two events:"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "outputs": [
+    {
+     "data": {
+      "text/plain": "2"
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "len(results['output']['inference_results'])"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Every event is represented as a dictionary with three fields:\n",
+    "\n",
+    "The event window, a list consisting of the start and end index values for the event. This describes when the event occurred.\n",
```

Comment:
Can we list these as 3 bullet points or add another newline between field descriptions?